pyanaconda/modules/storage/partitioning/automatic_partitioning.py

```diff
@@ -17,24 +17,90 @@
 #
 from blivet.devicelibs.crypto import MIN_CREATE_ENTROPY
 from blivet.partitioning import do_partitioning, grow_lvm
+from blivet.size import Size
 from blivet.static_data import luks_data
 
 from pyanaconda.anaconda_loggers import get_module_logger
+from pyanaconda.core.configuration.anaconda import conf
+from pyanaconda.core.configuration.storage import PartitioningType
 from pyanaconda.modules.common.structures.partitioning import PartitioningRequest
 from pyanaconda.modules.storage.partitioning.noninteractive_partitioning import \
     NonInteractivePartitioningTask
 from pyanaconda.modules.storage.partitioning.schedule import get_candidate_disks, \
     schedule_implicit_partitions, schedule_volumes, schedule_partitions
 from pyanaconda.platform import platform
-from pyanaconda.storage.partitioning import get_full_partitioning_requests, \
-    get_default_partitioning
+from pyanaconda.storage.partspec import PartSpec
 from pyanaconda.storage.utils import suggest_swap_size, get_pbkdf_args
 
 log = get_module_logger(__name__)
 
 __all__ = ["AutomaticPartitioningTask"]
 
 
+SERVER_PARTITIONING = [
+    PartSpec(
+        mountpoint="/",
+        size=Size("2GiB"),
+        max_size=Size("15GiB"),
+        grow=True,
+        btr=True,
+        lv=True,
+        thin=True,
+        encrypted=True
+    ),
+    PartSpec(
+        fstype="swap",
+        grow=False,
+        lv=True,
+        encrypted=True
+    )
+]
+
+WORKSTATION_PARTITIONING = [
+    PartSpec(
+        mountpoint="/",
+        size=Size("1GiB"),
+        max_size=Size("70GiB"),
+        grow=True,
+        btr=True,
+        lv=True,
+        thin=True,
+        encrypted=True),
+    PartSpec(
+        mountpoint="/home",
+        size=Size("500MiB"), grow=True,
+        required_space=Size("50GiB"),
+        btr=True,
+        lv=True,
+        thin=True,
+        encrypted=True),
+    PartSpec(
+        fstype="swap",
+        grow=False,
+        lv=True,
+        encrypted=True
+    )
+]
```

Comment:
I guess this will change to something which can be dynamically changed by a spin later. If that is correct, could you please add a comment with explanation of our plans?

Reply:
I don't understand. We already dynamically modify this partitioning based on the platform and the kickstart data. These are just default definitions.

Reply:
That's exactly what I asked for. To add comment that these are the default values and how they can be overridden from the outside. However if I look little bit lower where these are used then it make it's pretty self-explanation so you can ignore this.